opsfiles/templates/nginx/vhost/phanbook.conf

```diff
@@ -2,14 +2,15 @@ server {
     listen 80;
     server_name localhost;
 
-    application/x-font-ttf        ttc ttf;
-    application/x-font-otf        otf;
-    application/font-woff         woff;
-    application/font-woff2        woff2;
-    application/vnd.ms-fontobject eot;
-
+    
     index index.php index.html;
-    root '/usr/share/nginx/html/phanbook/public';
+
+    set $root_path '/usr/share/nginx/html/phanbook';
+    set $root_modules "${root_path}/core/modules";
+    set $root_public "${root_path}/public";
```

Comment:
Variables should not be used as template macros. Variables are evaluated in the run-time during the processing of each request, so they are rather costly compared to plain static configuration. Using variables to store static strings is also a bad idea. Instead, a macro expansion and "include" directives should be used to generate configs more easily and it can be done with the external tools, e.g. sed + make or any other common template mechanism.